model/dialogue/hmnet_model.py

```diff
@@ -0,0 +1,199 @@
+from ..base_model import SummModel
+import argparse
+import os
+import torch
+import gzip
+import json
+import uuid
+from nltk import word_tokenize
+import sys
+from ..third_party.HMNet.Models.Trainers.HMNetTrainer import HMNetTrainer
+from ..third_party.HMNet.Utils.Arguments import Arguments
+from ..third_party.HMNet.Models.Networks.MeetingNet_Transformer import MeetingNet_Transformer
+from ..third_party.HMNet.Models.Criteria.MLECriterion import MLECriterion
+
+# TODO: readme -- download the pretraining model,
+#       git clone HMNet before use,
+#       the dependencies that are needed to be installed.
+# TODO: set the role dict, spacy.
+
+
+class HMNetModel(SummModel):
+    # static variables
+    model_name = "HMNET"
+    is_extractive = False
+    is_neural = True
+
+    def __init__(self):
+        super(HMNetModel, self).__init__()
+        self.root_path = self._get_root()
+        self.opt = self._parse_args()
+        self.model = HMNetTrainer(self.opt)
+
+    def _get_root(self):
+        root_path = os.getcwd()
+        while 'model' not in os.listdir(root_path):
+            root_path = os.path.dirname(root_path)
+        root_path = os.path.join(root_path, 'model/dialogue')
+        return root_path
+
+    def _parse_args(self):
+        parser = argparse.ArgumentParser(description='HMNet: Pretrain or fine-tune models for HMNet model.')
+        parser.add_argument('--command', default='evaluate', help='Command: train/evaluate')
+        parser.add_argument('--conf_file',
+                            default=os.path.join(self.root_path, 'hmnet/config/dialogue_conf'),
```

Comment:
If you decide to change the config file name, remember to change it here also

Reply:
Thanks for the reminder!